pkg/resources/factory.go

```diff
@@ -103,6 +104,16 @@ func (rf *resourceFactory) GetResourcePool(rc *types.ResourceConfig, deviceList
 		}
 	}
 
+	// filter for rdma devices
+	if rc.IsRdma {
+		rdmaDevices := make([]types.PciNetDevice, 0)
+		for _, dev := range filteredDevice {
+			if dev.GetRdmaSpec().IsRdma() {
+				rdmaDevices = append(rdmaDevices, dev)
```

Comment:
Looks like ``rdmaDevices`` is created but not used anywhere, shall it be assigned back to ``filteredDevice``?

Reply:
fixed